packages/cli/src/util/deploy/get-deployments-by-project-id.ts

```diff
@@ -1,26 +1,47 @@
 import { URLSearchParams } from 'url';
-import { Alias } from '../../types';
+import { Deployment } from '../../types';
 import Client from '../client';
 
-type Response = {
-  deployments: DeploymentPartial[];
-};
-export interface DeploymentPartial {
-  id: string;
-  name: string;
-  url: string;
+type LegacyDeployment = {
+  aliasAssigned?: number | boolean | null;
+  aliasError?: {
+    code: string;
+    message: string;
+  } | null;
+  buildingAt: number;
+  checksConclusion?: 'succeeded' | 'failed' | 'skipped' | 'canceled';
+  checksState?: 'registered' | 'running' | 'completed';
   created: number;
-  createdAt: number;
-  aliases: Alias[];
+  createdAt?: number;
+  creator: {
+    uid: string;
+    email?: string;
+    username?: string;
+    githubLogin?: string;
+    gitlabLogin?: string;
+  };
+  inspectorUrl: string | null;
+  isRollbackCandidate?: boolean | null;
+  meta?: { [key: string]: string | undefined };
+  name: string;
+  ready?: number;
+  source?: 'cli' | 'git' | 'import' | 'import/repo' | 'clone/repo';
   state:
     | 'BUILDING'
     | 'ERROR'
     | 'INITIALIZING'
     | 'QUEUED'
     | 'READY'
     | 'CANCELED';
-  creator: { uid: string };
-}
+  target?: 'production' | 'staging' | null;
+  type: 'LAMBDAS';
+  uid: string;
+  url: string;
+};
```

Comment:
**suggestion (non-blocking):** extract general type

This type could be useful elsewhere. Can we make it more discoverable by placing it a common place instead?